### ISSUE TITLE
BehatCheck:checkBaseUrl:uri should better handle the missing HTTP protocol from the Drush URI check

### ISSUE THREAD

**Issue by anavarre** [acquia/blt#2695]
Consider this:

```
$ cat blt/local.blt.yml 
project:
  machine_name: mysite
  local:
    protocol: http
    hostname: '${project.machine_name}.local'
```

Upon running `blt setup:behat` the above configuration populates `tests/behat/local.yml` with the following:

```
Behat\MinkExtension:
  (snip)
  # configure the base url for your site
  base_url: http://mysite.local 
```

However, `blt doctor --site=mysite.local` will complain like so:

```
BehatCheck:checkBaseUrl:uri

base_url in tests/behat/local.yml does not match the site URI.

Behat base_url is set to http://mysite.local.
Drush site URI is set to mysite.local. 
```

To change this behavior, this requires editing `docroot/sites/mysite.local/local.drush.yml` such as instead of having:

```
options:
  uri: '${project.local.uri}'
```

We force the URI like so:

```
options:
  uri: 'http://mysite.local'
```

So ideally, what would need to happen is that `local.drush.yml` stays untouched and we make sure the correct URI is being returned...with the protocol, because the URI is otherwise indeed correct.

**Comment by anavarre** [comment]
Making visible to @mikemadison13

**Comment by codexmas** [comment]
Encountered this issue today as well.

**Comment by mikemadison13** [comment]
closing due to inactivity

**Comment by anavarre** [comment]
Hey @mikemadison13 it's not inactivity on our end ;-)

This is very much something I'd like to see fixed. What more do you need?

**Comment by mikemadison13** [comment]
^^ that. will keep it open. thx

**Comment by danepowell** [comment]
Wouldn't this be fixed by changing `local.drush.yml` from:
```
options:
  uri: '${project.local.uri}'
```
to:
```
options:
  uri: '${project.local.protocol}://${project.local.uri}'
```

If that works it's a pretty simple change to default.local.drush.yml.

**Comment by danepowell** [comment]
Sorry, I misread that configuration value. `${project.local.uri}` _already_ includes the protocol (for some reason I confused it with `${project.local.hostname}`). So this should already be fixed in the latest release of 9.2.x and 10.x.

@anavarre it looks like your `local.drush.yml` was generated incorrectly, or by an older version of BLT and never updated. If you just change the uri to include the hostname (which happens by default for new projects) this should be fixed.

If you are able to reproduce this on a newly-created project, please open a new issue and provide steps to reproduce (starting from `composer create-project acquia/blt-project...`). Thanks.
